delivery.yaml

```diff
@@ -0,0 +1,62 @@
+build_steps:
+    - desc: 'Install Docker'
+      cmd: |
+        # enable experimental features (--squash)
+        mkdir -p /etc/systemd/system/docker.service.d
+        cat << EOF > /etc/systemd/system/docker.service.d/docker.conf
+        [Service]
+        ExecStart=
```

Comment:
*gulp* i'm not familiar with CDP config yet....but i this correct? two times `ExecStart` ?